docs/security.md

```diff
@@ -100,12 +118,23 @@ authorization:
   admin: 'lb_admin'
   user: 'lb_user'
   api: "lb_api"
-  ldapHost:
-  ldapPort:
-  ldapBindDn:
-  ldapSearch:
-  ldapPassword:
+  ldapConfigPath: "<ldap_config_path>"
+```
+
+The ldap config file should have the following contents
+
 ```
+  ldapHost: '<ldap sever>'
+  ldapPort: <port>
+  useTls: <true/false>
+  useSsl: <true/false>
+  ldapAdminBindDn: <>
+  ldapUserBaseDn: <>
+  ldapUserSearch: <>
+  ldapGroupMemberAttribute: <>
+  ldapAdminPassword: <>
+```
```

Comment:
kinda duplicate of what i suggested in other [PR](https://github.com/trinodb/trino-gateway/pull/32/commits/a11a30d7ef9e8dfcbd3e34648f66d40cf092d781#diff-2c506cc756a5333f8fb5acb850d443980706602048f9ddb3dc515abacbfbce6a) but anyway, 
aren't these configs `ldapTrustStorePath` and `ldapTrustStorePassword` conf necessary if using either TLS/SSL? 
I think it's rare that user doesn't use either one so how about adding this confs?

Reply:
Did your PR get merged?
Yes, if you are using internal issued certs then you will need to use those specific paths.

Reply:
Is the other PR merged?

Reply:
Add those extra properties. somehow I missed those in the first update.